eel/examples/node/cli.rs

```diff
@@ -182,3 +191,32 @@ fn pay_invoice<'a>(
 
     Ok(())
 }
+
+fn list_payments(node: &LightningNode) -> Result<(), String> {
+    let payments = match node.get_latest_payments(100) {
+        Ok(p) => p,
+        Err(e) => return Err(e.to_string()),
+    };
+
+    println!("Total of {} payments\n", payments.len());
```

Comment:
What information are you wanting to log here, if the payments length is hardcoded to be `100` ?

Reply:
The idea was to provide the user of the CLI with the number of payments that will be printed immediately after. But we can improve the example node for this command to accept the number of payments, making the limit more understandable.

Reply:
Yes maybe later, I think it is good enough for now.